frontend/components/CardItemComponent/index.jsx

```diff
@@ -37,7 +37,7 @@ const CardItem = ({ data }) => {
               btnSize="large"
             >
               <span className="mr-4">
-                <img className="w-8" src="/static/images/github.png" />
+                <img loading = "lazy" className="w-8" src="/static/images/github.png" />
```

Comment:
Nice Job Abdullrahman, 
But I recommend to read about the behavior of "loading lazy"  on different browsers, I recommend to use a third library to implement lazyload like [vanilla-lazyload](https://github.com/verlok/vanilla-lazyload) its almost 2.4kB only 

or we could accept the current implementation

Reply:
I will consider checking it.